docs/hippocampus_model.md

```diff
@@ -0,0 +1,35 @@
+
+## Terms of use
+Please note that this model is intended for research purposes only. You are responsible for the performance, the necessary testing, and if needed any regulatory clearance for any of the models produced by this toolbox.
+
+---
+
+## Usage
+The following instructions assume you have completed the preceding setup steps in the [InnerEye README](https://github.com/microsoft/InnerEye-DeepLearning/), in particular, [Setting up Azure Machine Learning](https://github.com/microsoft/InnerEye-DeepLearning/blob/main/docs/setting_up_aml.md).
+
+### Create an Azure ML Dataset
+To evaluate this model on your own data, you will first need to register an [Azure ML Dataset](https://docs.microsoft.com/en-us/azure/machine-learning/v1/how-to-create-register-datasets). You can follow the instructions in the InnerEye repo for [creating datasets](https://github.com/microsoft/InnerEye-DeepLearning/blob/main/docs/creating_dataset.md) in order to do this.
+
+## Downloading the model
+The saved weights from the trained Hippocampus model can be downloaded along with the source code used to train it from [our github releases page](https://github.com/microsoft/hi-ml/releases).
+
+### Registering a model in Azure ML
+To evaluate the model in Azure ML, you must first [resgister an Azure ML Model](https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.core.model.model?view=azure-ml-py#remarks). To register the Hippocampus model in your AML Workspace, unpack the source code downloaded in the previous step and follow the instructions to run InnerEye's [move_model.py](https://github.com/microsoft/InnerEye-DeepLearning/blob/main/docs/move_model.md) script in the `-a=upload` direction.
+
+i.e.
+```shell
+python InnerEye/Scripts/move_model.py -a upload --path ./ --workspace_name <workspace_name> --resource_group <resource_group> --subscription_id <subscription_id> --model_id Hippocampus:111
+```
+### Evaluating the model
+You can evaluate the model either in Azure ML  or locally using the downloaded checkpoint files. These 2 scenarios are described in more detail, along with instructions in  [testing an existing model](https://github.com/microsoft/InnerEye-DeepLearning/blob/main/docs/building_models.md#testing-an-existing-model).
+
+For example, to evalute the model on your Dataset in Azure ML:
+```shell
+python InnerEye/ML/runner.py --azure_dataset_id=<my dataset id> --model=Hippocampus --model_id=Hippocampus:111 --experiment_name evaluate_hippocampus_model --azureml --no-train
```

Comment:
Maybe this can be formatted as in my suggestion above, if you think it made it more readable.

Reply:
Yes looks good. Have updated in the same format